reference/src/layout/unions.md

```diff
@@ -6,17 +6,98 @@ not to change until an RFC ratifies them.
 
 [#13]: https://github.com/rust-rfcs/unsafe-code-guidelines/issues/13
 
-The only degree of freedom the compiler has when computing the layout of a union
-like
+### Layout of individual union fields
 
-```rust,ignore
-union U { f1: T1, f2: T2 }
+The layout of each union field is determined by its type. 
```

Comment:
I think this is basically https://github.com/rust-lang/unsafe-code-guidelines/issues/185 for unions.

Reply:
By this I mean: I think the text here should use that picture. The rationale already explains why that picture is accurate.

"The layout of each union field is determined by its type" sounds like a tautology. The layout of all fields is already given when we compute the layout of a union.